man/rules/meson.build

```diff
@@ -6,6 +6,7 @@ manpages = [
  ['busctl', '1', [], ''],
  ['coredump.conf', '5', ['coredump.conf.d'], 'ENABLE_COREDUMP'],
  ['coredumpctl', '1', [], 'ENABLE_COREDUMP'],
+ ['pstore.conf', '5', ['pstore.conf.d'], 'ENABLE_PSTORE'],
```

Comment:
The file has
```
# Do not edit. Generated by make-man-rules.py.
```
at the top. Please, run `ninja -C build man/update-man-rules` to regenerate this file properly.

Reply:
Regenerated.